10.Mender-server/docs.md

```diff
@@ -2827,6 +2827,12 @@ New changes in workflows-enterprise since 2.3.0:
 
 _Released 09.25.2022_
 
+### Security fixes
+
+* Fixed CVE-2022-41324 - Read-only API access can leak secrets in Mender.
+  See the official announcement for more information:
+  https://mender.io/blog/cve-2022-45929-cve-2022-41324-improper-access-control-for-low-privileged-users
```

Comment:
This fix was not released in this version. Please remove

Reply:
Okay, please see comments in ticket: https://northerntech.atlassian.net/browse/SEC-591